README.md

```diff
@@ -1 +1,76 @@
-# kanban-board
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Leaderboard App
+
+> This is an app that allows users to add theirs scores to the Leaderboard API.
+
+## Built With
+
+- **Languages:**
+  - HTML5 & CSS3
+  - JavaScript
+
+- **Preprocessors:**
+  - Css
+  - Tailwind
+
+- **Framework:**
+  - React
+  - Next js
+
+- **Conventions and Methodologies:**
+  - TTD
+  - MVP pattern
+
```

Comment:
You must have used your leaderboard project's README here.

- [ ] Kindly edit this README to have documentation relating to the Math Magician app. 👍